app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,45 @@
+/* universal */
+body {
+padding-top: 30px;
+}
```

```suggestion
body {
  padding-top: 30px;
}
```
sai indent, self check file này